src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Compiler/Resources/Imports.targets

```diff
@@ -0,0 +1,13 @@
+﻿<!--
+ Copyright (c) .NET Foundation. All rights reserved.
+ Licensed under the Apache License, Version 2.0.
+ -->
+
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Condition=" '$(TargetFramework)' == '' AND '$(ExcludeRestorePackageImports)' != 'true' ">
+    <Import Project="$(NuGetPackageRoot)microsoft.visualstudio.web.codegeneration.tools\{0}\buildCrossTargeting\microsoft.visualstudio.web.codegeneration.tools.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.visualstudio.web.codegeneration.tools\{0}\buildCrossTargeting\microsoft.visualstudio.web.codegeneration.tools.targets')" />
+  </ImportGroup>
+  <ImportGroup Condition=" '$(TargetFramework)' != '' AND '$(ExcludeRestorePackageImports)' != 'true' ">
+    <Import Project="$(NuGetPackageRoot)microsoft.visualstudio.web.codegeneration.tools\{0}\build\microsoft.visualstudio.web.codegeneration.tools.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.visualstudio.web.codegeneration.tools\{0}\build\microsoft.visualstudio.web.codegeneration.tools.targets')" />
```

Comment:
You can use an MSBuild variable instead of needing to `string.Format` this entire file.

Example:
```xml
<Import Project="$(_CodeGenTargetsLocation)\microsoft.visualstudio.web.codegeneration.tools.targets" Condition="Exists('$(_CodeGenTargetsLocation)\microsoft.visualstudio.web.codegeneration.tools.targets" />
```

Has the added benefit of making testing easier as you don't you can import from an arbitrary folder.